app/components/profile/api_token_card_component.rb

```diff
@@ -0,0 +1,10 @@
+class Profile::APITokenCardComponent < ApplicationComponent
+  def initialize(api_token: nil, token: nil)
+    @api_token = api_token
+    @token = token
+  end
+
```

Comment:
pour jarter de la logique dans la vue ? (peut être pas ouf)

```suggestion
def api_and_packed_tokens
  current_administrateur.api_tokens.map do |api_token|
    if api_token == @created_api_token
      [api_token, @created_packed_token]
    else
      [api_token, nil]
  end
end
```